pulpcore/cli/deb/context.py

```diff
@@ -119,6 +120,6 @@ class PulpAptRepositoryContext(PulpRepositoryContext):
     DELETE_ID = "repositories_deb_apt_delete"
     SYNC_ID = "repositories_deb_apt_sync"
     VERSION_CONTEXT = PulpAptRepositoryVersionContext
-
+    CAPABILITIES = {"pulpexport": [PluginRequirement("deb")]}
```

Comment:
Most probable this needs a versioned requirement.

Reply:
I updated to require version 2.20.0 based on https://github.com/pulp/pulp_deb/releases/tag/2.20.0 (https://github.com/pulp/pulp_deb/commit/fdebfdae981eabffed03cd9d4a11ad41bef11a8b).